fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/socialmedia/SocialMediaViewModel.java

```diff
@@ -68,11 +111,86 @@ public void openSend() {
     mOpenSendEvent.postValue(new SingleEvent<>(true));
   }
 
+  public void openSearch() {
+    mOpenSearchEvent.postValue(new SingleEvent<>(true));
+  }
+
   public void openFollowing() {
     mOpenFollowingEvent.postValue(new SingleEvent<>(true));
   }
 
   public void openProfile() {
     mOpenProfileEvent.postValue(new SingleEvent<>(true));
   }
+
+  public void sendNewChirpEvent() {
+    mSendNewChirpEvent.postValue(new SingleEvent<>(true));
+  }
+
+  public void setNumberCharsLeft(Integer numberChars) {
+    mNumberCharsLeft.setValue(numberChars);
+  }
+
+  public void setLaoId(String laoId) {
+    mLaoId.setValue(laoId);
+  }
+
+  /** Subscribe to the channel: /root/<lao_id>/social/chirps */
+  public void subscribeToGeneralChannel(String laoId) {
+    Log.d(TAG, "subscribing to channel: /root/" + laoId + "/social/chirps");
+
+    String channel = "/root/" + laoId + "/social";
```

Comment:
The comments and the channel are different.
Should it be .../social/chirps or only .../social ?

Reply:
I forgot to change it from another branch, it should be .../social/chirps